src/views/Beneficiaries/index.tsx

```diff
@@ -117,18 +121,18 @@ const Beneficiaries: React.FC<{ isLoading?: boolean }> = props => {
                     <Tabs defaultIndex={+getByKey('state')}>
                         <TabList>
                             <Tab
-                                number={data?.beneficiaryEntities?.filter((elem: any) => elem.state === 0)?.length}
-                                onClick={() => update({ 'page': 0, 'state': 0})}
+                                number={beneficiariesStateLength(0)}
+                                onClick={() => update('state', 0)}
```

Comment:
Don't remove the reset of the page, otherwise when changing between tabs the page won't change, and then if we have 10 pages on tab 0, and only 1 in tab 2, if we are on page 5 in tab 0 and we change to the tab 1, we'll stay in the page 5 which as no results. Whenever we change the tab we need to ensure we go to the first page of that tab.